hermes-management/src/test/groovy/pl/allegro/tech/hermes/management/domain/group/GroupServiceSpec.groovy

```diff
@@ -0,0 +1,60 @@
+package pl.allegro.tech.hermes.management.domain.group
+
+import pl.allegro.tech.hermes.api.Group
+import pl.allegro.tech.hermes.api.PatchData
+import pl.allegro.tech.hermes.api.helpers.Patch
+import pl.allegro.tech.hermes.domain.group.GroupRepository
+import pl.allegro.tech.hermes.management.domain.Auditor
+import spock.lang.Specification
+
+import java.security.Principal
+
+class GroupServiceSpec extends Specification {
+
+    static Group TEST_GROUP = new Group("testGroup", "testOwner", "testTeam", "testContact")
```

Comment:
use the builders (`GroupBuilder`) -> then you can probably skip this static and just use `group('testGroup').build()` inside methods